restapi/src/main/java/io/stargate/web/docsapi/service/DocumentService.java

```diff
@@ -616,74 +616,19 @@ public void deleteAtPath(
     db.delete(keyspace, collection, id, convertedPath, now);
   }
 
-  public JsonNode searchDocuments(
```

Comment:
I removed this, as it's dead code and we are now moving far far away from it (nothing uses it)